app/models/application_form.rb

```diff
@@ -246,6 +246,10 @@ def can_add_more_choices?
     choices_left_to_make.positive?
   end
 
+  def recruited?
+    application_choices.recruited.any?
+  end
```

Comment:
i appreciate it's fairly trivial but would you mind adding a unit test for this?

Reply:
have added!

Reply:
mille grazie